internal/modules/oracle/test.py

```diff
@@ -48,29 +60,56 @@ def test_query_oracle_params(self):
     # test_prevotes tests to make sure that we can submit prevotes
     def test_prevotes(self):
         # Get Hash
-        vote_hash_1 = get_hash(STATIC_EXCHANGE_RATES, STATIC_SALT, validator1_acc["address"])
-        vote_hash_2 = get_hash(STATIC_EXCHANGE_RATES, STATIC_SALT, validator2_acc["address"])
+        vote_hash_1 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator1_acc["address"])
+        vote_hash_2 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator2_acc["address"])
         # Submit 1st prevote
         status = tx_submit_prevote(validator1_acc["name"], vote_hash_1, validator1_home)
         self.assertTrue(status)
-        time.sleep(1)
+
         # Submit 2nd prevote
         status = tx_submit_prevote(validator2_acc["name"], vote_hash_2, validator2_home)
         self.assertTrue(status)
-        time.sleep(1)
 
         # Query to verify prevotes exist
         status, prevote_1 = query_aggregate_prevote(validator1_acc["address"])
         self.assertTrue(status)
-        self.assertEqual(prevote_1["hash"], vote_hash_1)
+        self.assertEqual(prevote_1["aggregate_prevote"]["hash"].upper(), vote_hash_1)
         status, prevote_2 = query_aggregate_prevote(validator2_acc["address"])
         self.assertTrue(status)
-        self.assertEqual(prevote_2["hash"], vote_hash_2)
+        self.assertEqual(prevote_2["aggregate_prevote"]["hash"].upper(), vote_hash_2)
+
+    # test_votes tests to make sure that we can submit votes
+    def test_votes(self):
+        # Get Hash
+        vote_hash = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator3_acc["address"])
+
+        # Submit prevote
+        tx_submit_prevote(validator3_acc["name"], vote_hash, validator3_home)
```

Comment:
So test_votes only runs the prevote + vote for one validator? We may want to expand this later on

Reply:
I'm down to add another validator back in, or use all 3! the initial goal was to get one working. Using validator 3 has the added benefit of not having to wait for the next voting period to submit prevotes or else you have to put a ~30 seconds sleep in. (because the pre vote test runs right before this one)

Reply:
I don't think 2 validators submitting votes at the same time will be possible until we make this thing multi-threaded or make clever use of broadcast_mode.